app-emulation/virt-firmware/files/no_shim_check.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/virt/firmware/bootcfg/main.py b/virt/firmware/bootcfg/main.py
+index 7294efa..32302da 100644
+--- a/virt/firmware/bootcfg/main.py
++++ b/virt/firmware/bootcfg/main.py
+@@ -80,9 +80,6 @@ def create_boot_entry(efiuki, options):
+ 
+ 
+ def add_uki(cfg, options):
+-    if not options.shim and not firmware_loads_efi_binary(cfg, options.adduki):
```

Comment:
This second condition should cause shim to not be required if the signing key is already registered in the firmware.